src/QMCDrivers/DMC/DMCBatched.cpp

```diff
@@ -429,6 +432,14 @@ bool DMCBatched::run()
   IndexType num_blocks = qmcdriver_input_.get_max_blocks();
 
   estimator_manager_->startDriverRun();
+
+  //start walker log manager
+  wlog_manager_ = std::make_unique<WalkerLogManager>(walker_logs_input, allow_walker_logs, get_root_name(), myComm);
+  std::vector<WalkerLogCollector*> wlog_collectors;
+  for (auto& c: crowds_)
+    wlog_collectors.push_back(&c->wlog_collector_);
+  wlog_manager_->startRun(wlog_collectors);
```

Comment:
a vector of raw pointers is a big no. Crowds own collectors and thus, RefVector is a better way of expressing the ownership.
```
RefVector<<WalkerLogCollector> wlog_collectors;
```

Reply:
Also it is better to make wlog_manager_ accepts wlog_collectors via constructor and save it. Then later writeBuffers doesn't need arguments.

Reply:
This change can be a bit involving. Defer it to a later PR.

Reply:
(Nothing to change here)